### ISSUE TITLE
CSV to Neo4j failing on codon

### ISSUE THREAD

**Issue by jamesamcl** [EBISPOT/ols4#119]
Lots of errors like this

```
/obo/DRON_00028160","http://purl.obolibrary.org/obo/DRON
  original error: Tried to read a field larger than buffer size 4194304. A common cause of this is that a field has an unterminated quote and so will try to seek until the next quote, which ever line it may be on. This should not happen if multi-line fields are disabled, given that the fields contains no new-line characters. This field started at /nfs/production/parkinso/spot/ols4/dev/./output_csv/dron_classes.csv:662021

WARNING Import failed. The store files in /nfs/production/parkinso/spot/ols4/dev/neo4j/data/databases/neo4j are left as they are, although they are likely in an unusable state. Starting a database on these store files will likely fail or observe inconsistent records so start at your own risk or delete the store manually
org.neo4j.internal.batchimport.input.InputException: ERROR in input
  data source: BufferedCharSeeker[source:/nfs/production/parkinso/spot/ols4/dev/./output_csv/dron_classes.csv, position:1736441856, line:662021]
  in field: _json:string:31
  for header: [id:ID, :LABEL, isPreferredRoot:string[], http__//purl.obolibrary.org/obo/chebi/inchi:string[], isDefiningOntology:string[], isRoot:string[], hasChildren:string[], http__//purl.obolibrary.org/obo/chebi/monoisotopicmass:string[], type:string[], relatedTo:string[], http__//www.geneontology.org
```

**Comment by jamesamcl** [comment]
There is a linebreak in the CSV

```
"dron+class+http://purl.obolibrary.org/obo/OBI_0000011","OntologyEntity|OntologyClass","false","","false","false","true","","class|entity","","","","http://purl.obolibrary.org/obo/BFO_0000
objectives is a planned process.","","Bjoern Peters","http://purl.obolibrary.org/obo/IAO_0000122","A process that realizes a plan which is the concretization of a plan specification.","","
```

might be related

**Comment by jamesamcl** [comment]
neo4j-admin import has this option

      --multiline-fields[=<true/false>]
                             Whether or not fields from input source can span
                               multiple lines, i.e. contain newline characters.
                               Default: false

But we already set it to true.

**Comment by jamesamcl** [comment]
It's fine to drop all of the newlines in Neo4j and even set `--multiline-fields=false` (which apparently makes the import faster) as we never display the Neo4j fields anyway in the APIs or frontend; they are just for querying.

However, the newline is inside the `_json` field, which begs the question why it isn't using `\n`.

In `ontologies.json`, `\n` is used, so json2neo turns it into an actual newline somehow when printing the csv.

**Comment by jamesamcl** [comment]
Ok so there's actually no issue with having newlines in the CSV. It's just that the length of the row exceeded the default buffer size of neo4j-admin import. Fixed by adding `--read-buffer-size=16777216` (16 mb).
